### DISCUSSION TITLE
Use defined symbol in PostActions

### DISCUSSION THREAD

**Discussion by lucasneuwit** [dotnet/templating#4525] [General]
Hello! Is there a way of using a defined symbol as a reference in a post-action? More specifically, what I am trying to accomplish is the following:

```json
"symbols": {
        "EntityName": {
            "type": "parameter",
            "datatype": "text",
            "replaces": "EntityName",
            "fileRename": "EntityName",
            "description": "The name of the entity to be migrated.",
            "isRequired": true
        },
    },
"postActions": [{
    "actionId":"3A7C4B45-1F5D-4A30-959A-51B88E82B5D2",
    "args": {
        "executable": "my-script.cmd",
        "args": "EntityName"
    }
}]
```

Where EntityName in args refers to the previously declared symbol.

Thanks in advance for your time!

**Comment by vlada-shubina** [comment]
Unfortunately, not. We have a feature for this here: https://github.com/dotnet/templating/issues/1600. 

The possible workaround is mentioned here: https://github.com/dotnet/templating/issues/1600#issuecomment-412112022, you may define a variable instead of argument inside your script, which gets replaced when template content is created and then run script without args.
